### ISSUE TITLE
Normal / Layout View Mode Blending

### ISSUE THREAD

**Issue by Brittank88** [absoIute/Mega-Hack-Pro-Future#410]
Similar to the effect seen [here](https://youtu.be/ZaTGx5ubzH8?t=416).
This is a clean (succmydecc-less) version of #407.

**Comment by Nicko2102** [comment]
oh thanks :)

> This is a clean (succmydecc-less) version of #407.

exactly what this needed, thank you

**Comment by Brittank88** [comment]
> 

He has been banned from the tracker, for being continually disruptive, btw.

**Comment by Nicko2102** [comment]
wait you mean the entire repo? like the suggestions part?

**Comment by Brittank88** [comment]
> wait you mean the entire repo? like the suggestions part?

Yep, he's gone. Both me and Absolute have had enough. Disruption, entitlement, rudeness to the point of intentionally being difficult, etc. That sort of behaviour is not welcome here.

**Comment by Nicko2102** [comment]
> Yep, he's gone. Both me and Absolute have had enough. Disruption, entitlement, rudeness to the point of intentionally being difficult, etc. That sort of behaviour is not welcome here.

Thank you so much, i was about 2 stuck up comments away from contacting you on discord, it's satisfying to finally see him gone

**Comment by Brittank88** [comment]
> > Yep, he's gone. Both me and Absolute have had enough. Disruption, entitlement, rudeness to the point of intentionally being difficult, etc. That sort of behaviour is not welcome here.
> 
> Thank you so much, i was about 2 stuck up comments away from contacting you on discord, it's satisfying to finally see him gone

He literally tried to contact us on one of [my repositories](https://github.com/Brittank88/Brittank88.github.io/issues/3) that was completely unrelated to this project. Absolutely insufferable. We both have him blocked now which means he can't interact with either of our repositories or mention us.

**Comment by Nicko2102** [comment]
> Also we should probably stay on-topic lmao, just thought I'd provide some context/info.

appreciate it

**Comment by insufferablebeing** [comment]
> Thank you so much, i was about 2 stuck up comments away from contacting you on discord, it's satisfying to finally see him gone
im sorry
